src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/sidepanel.jelly

```diff
@@ -22,4 +22,9 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<st:include it="${it.primaryView}" page="sidepanel.jelly" xmlns:st="jelly:stapler"/>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <j:if test="${view == null}"> <!-- true when rendering from ModelObjectWithContextMenu.ContextMenu.from -->
```

Comment:
Not really, it is just going directly to the object’s `sidepanel` view, whereas a regular folder index page load actually loads `View/sidepanel.jelly` via [this trick](https://github.com/jenkinsci/cloudbees-folder-plugin/blob/5f239712601807f3f1eb6f573a8fa550261978a7/src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java#L416-L422).